code/modules/mob/mob_movement.dm

```diff
@@ -509,7 +515,11 @@
 /mob/proc/zMove(dir, feedback = FALSE)
 	if(dir != UP && dir != DOWN)
 		return FALSE
+	var/turf/source = get_turf(src)
```

Comment:
isn't loc being a turf a prerequisite for zMove being called

Reply:
Someone could be calling up() from inside a locker, bluespace body bag, or something else dumb no?

Reply:
I don't think they can